web/templates/_menu.jinja2

```diff
@@ -4,10 +4,10 @@
 
     <div class="view" id="view-load">
         <h5 class="title">
-            Enter a <b>bus route</b> relation ID
+            Enter a <b>bus route</b> relation ID or URL
```

Comment:
```suggestion
            Enter a <b>bus route</b> relation ID
```

After considering my initial suggestion, if the non-numeric input is removed, it will always become an ID. Additionally, I am hesitant about adding more text in the first menu panel. Therefore, for now, I believe it would be a better option to not explicitly mention the URL support.